docs/installation.rst

```diff
@@ -117,6 +117,70 @@ the old version using the same steps as shown above.
+| Linux PowerPC     | Yes [2]_ |          |           |            |
++-------------------+----------+          |           |            |
+| Linux ARM         | Manually |          |           |            |
++-------------------+----------+----------+-----------+------------+
+| Mac OS X          | Yes [1]_ | Yes      | Yes       | Yes (all)  |
++-------------------+----------+----------+-----------+            |
+| FreeBSD           | Yes [2]_ | Yes      | Yes       |            |
++-------------------+----------+----------+-----------+            |
+| OpenBSD           | Yes [2]_ | n/a      | n/a       |            |
++-------------------+----------+----------+-----------+            |
+| NetBSD            | Manually | No [5]_  | No [7]_   |            |
++-------------------+----------+----------+           +------------+
+| Solaris 11        | Yes [2]_ | No       |           | n/a        |
++-------------------+----------+----------+           +------------+
+| OpenIndiana       | No       | No       |           | n/a        |
++-------------------+----------+----------+-----------+------------+
+| Windows (native)  | Yes [3]_ | Yes      | ?         | ?          |
++-------------------+----------+----------+-----------+------------+
+| Windows (cygwin)  | Manually | No [4]_  | No        | No         |
++-------------------+----------+----------+-----------+------------+
+
+Some Distributions (e.g. Debian) run additional tests after each release, these
+are not reflected here.
+
+Other Unix-like operating systems may work as well, but have not been tested at all.
+
+Note that most of the platfrom-dependent features also depend on the file system.
+For example, ntfs-3g on Linux isn't able to convey NTFS ACLs.
+
+.. [1] `Travis-CI <https://travis-ci.org/borgbackup/borg>`_
+.. [2] `Golem <https://golem.enkore.de/view/Borg/>`_:
+    FreeBSD, OpenBSD and Solaris 11 are x64.
+    PowerPC runs on PowerMac G5 hardware.
+.. [3] `AppVeyor <https://ci.appveyor.com/project/borgbackup/borg/>`_,
+    Borg for Windows is under active development, see :issue:`983`
+.. [4] Cygwin tries to map NTFS ACLs to permissions with varying degress of success.
+.. [5] Feature request :issue:`433`
+.. [6] Only "nodump", "immutable", "compressed" and "append" are supported.
+    Feature request :issue:`618` for more flags.
+.. [7] Feature request :issue:`1332`
+
+.. [#tst]
+    *Manually* means that the test suite is run manually e.g. before a release -
+    but may be run even less frequently than that.
+.. [#acls] The native access control list mechanism of the OS. This normally limits access to
+    non-native ACLs. For example, NTFS ACLs aren't completely accessible on Linux with ntfs-3g.
+.. [#xattr] extended attributes; key-value pairs attached to a file, mainly used by the OS.
+    This includes resource forks on Mac OS X.
+.. [#flags] aka *BSD flags*. The Linux set of flags [6]_ is portable across platforms.
+    The BSDs define additional flags.
+
```

Comment:
As this is not just about platforms, but also feature support, this maybe should be linked together or merged with the other paragraph that talks about which fs features we support.